src/librustdoc/clean/types.rs

```diff
@@ -986,17 +1005,10 @@ impl Attributes {
             }
         }
 
-        let inner_docs = attrs
-            .iter()
-            .find(|a| a.doc_str().is_some())
-            .map_or(true, |a| a.style == AttrStyle::Inner);
-
         Attributes {
             doc_strings,
```

Comment:
Would it possible to also calculate this on demand? I think `extract_include` reads from the filesystem, so you probably want to add a cache - I'm fine with putting that off for a later PR.

Reply:
It had been moved as an `AttributeExt` method